lib/sles4sap/ipaddr2.pm

```diff
@@ -1480,7 +1482,7 @@ sub ipaddr2_configure_web_server {
     my (%args) = @_;
     croak("Argument < id > missing") unless $args{id};
     my @nginx_cmds = (
```

Comment:
Not for this PR but I would suggest not to loop the list of those commands as they have very different max runtime. 
For example `echo` does not need same 240s timeout as `zypper install`.

Another option could be storing each command entry as a hash with arguments to tweak arguments:
```
my %nginx_cmds = ({cmd=>'zypper install lolcat', timoeut=>240}, {cmd=>'echo "nyancat"', timeout=>10})
```

Reply:
The point is if 'echo' (or any other short command) gets stuck or any reason test will wait  4 min before failing instead of couple of seconds. 
It's about code efficiency.